chore: root README_ZH.md

mr-kelly opened the pull request:
Submit a pull request for this project.

# Why? 
Compatible for gitee


# What?
Add README_ZH.md

mr-kelly committed:
chore: root README_ZH

arucil commented on README_ZH.md:
```diff
@@ -0,0 +1,387 @@
+- **后端开发者**: 有 NestJS、TypeScript、Spring Boot、Java、SQL、Kubernetes、Terraform 的工作经验。并喜欢写高质量的代码，带有清晰的文档和单元测试。
+- **前端开发者**: 有 React、NextJS、TypeScript、WebPack 的工作经验。并喜欢写高质量的代码，带有清晰的文档和单元测试。
+
+无论时间和条件如何，如果你想要加入 APITable 团队，请不要犹豫，将您的简历发送到 talent@apitable
```
talent@apitable.com？

kwp-lab committed:
docs: rewrite contributor section for readme_ZH.md (#314)

Submit a pull request for this project.

<!-- If you have an Issue that related to this Pull Request, you can
copy this Issue's description -->

# What?

Rewrite the Contributor section for readme_ZH.md

kwp-lab committed:
Merge branch 'develop' into docs/readme_zhmd-for-simplified-chinese

kwp-lab committed:
docs: fix a mistake in README_ZH.md and sync to zh-CN/README.md

kwp-lab committed:
Merge branch 'develop' into docs/readme_zhmd-for-simplified-chinese

kwp-lab committed:
Merge branch 'develop' into docs/readme_zhmd-for-simplified-chinese
